portfolio.css

```diff
@@ -0,0 +1,97 @@
+body {
+    /* uiGradient - Vanusa */
+    background: #da4453; /* fallback for old browsers */
+    background: -webkit-linear-gradient(to right, #89216b, #da4453) fixed; /* Chrome 10-25, Safari 5.1-6 */
+    background: linear-gradient(to right, #89216b,     #da4453) fixed; /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
+}
+
+.col-4 {
+    position: relative;
+    float: left;
+    width: 25%;
+    height: 300px;
+}
+
+.gallery {
+    overflow: auto;
+    width: 1200px;
+    margin: 0 auto;
+}
+
+.portfolio-image {
+    height: 200px;
+    width: 200px;
+    border-radius: 20%;
+    margin: 0 auto;
+    background-size: contain;
+
+    /* Vertically and horizontally centre div */
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
+
+.icons {
+    margin-top: 70px;
+    text-align: center;
+}
+
+.ie .gallery img {
+    top: 0;
+    left: 0;
+}
+
+/****************************
+Media Queries
+/****************************/
+
+@media screen and (max-width: 1340px) {
+
+    .col-4 {
+        width: 33.3%;
+        margin-top: 10px;
+        margin: 0 auto;
```

Comment:
this is overriding the previous margin top declaration

Reply:
fixed